docs/Releasing-the-plugin.md

```diff
@@ -0,0 +1,25 @@
+# Releasing the performance plugin
+
+This document describes the steps to release the Performance plugin.
+
+## Preparing the release
+
+> The following changes need to be made through a pull request.
+
+### Update the version number
+
+The version number needs to be updated in the following files:
+
+- package.json
+- load.php
+- readme.txt
+
+### Update readme.txt
+
+Run `npm run readme -- -m "X.Y.Z"` to update the readme.txt file with the release changelog, with X.Y.Z as the release milestone name.
+
```

Comment:
Related to my comment on top of the file, I think here, i.e. at the very end of the preparation steps, we can explain the next step of a PR, e.g.
```suggestion

### Open a pull request

Push your local release branch to the GitHub repository and open a pull request against `trunk`. Make sure to tag at least 2 plugin maintainers to request a review. Once the pull request is approved by at least 2 plugin maintainers, the pull request can be merged.

If this is a major or minor release, please keep the release branch around since that branch can be used to later create patch releases from the same state of the codebase.

```